chrysler_pacifica_ehybrid_s_2019/docker-compose.yml

```diff
@@ -30,7 +30,7 @@ services:
     command: roscore
 
   platform:
-    image: usdotfhwastol/carma-platform:3.2.0
+    image: usdotfhwastol/carma-platform:urus
```

Comment:
I might not be familiar with the current process, but don't we want these to match the system release version CARMASystem_3.3.0 before we merge to master?

Reply:
Not wrong, but this PR is just to merge the release notes into the release branch. Was going to update the carma-platform image last, as we're still waiting to merge that one with master and as such there isn't a carma-platform:CARMASystem_3.3.0 image yet

Reply:
But this PR doesn't include any release notes? I guess I'm not seeing why this is necessary. I'll let @maefromm weigh in.

Reply:
My bad, confused it with release notes PR for CARMAPlatform. Will add docker-compose.yml updates now.

Reply:
@msmcconnell you're right, this would be the system release and all tags need to be updated to 3.3.0. @jtbaird please update all to say that if there's no more changes. And wait for the final approval from client. I will need to aprpove this.